website/src/documentation/runtime/environments.md

```diff
@@ -166,20 +166,18 @@ To change the repository, set the `docker-repository-root` option to a new locat
 
 After [building a container image](#building-container-images), you can store it in a remote Docker repository.
 
-The following steps push a Python SDK image to the [`docker-root-repository` value](#overriding-default-docker-targets).
+The following steps push a Python2.7 SDK image to the [`docker-root-repository` value](#overriding-default-docker-targets). 
+Please log in to the destination repository as needed. 
 
-1. Sign in to your Docker registry:
 ```
-docker login
+Upload it to the remote repository:
 ```
-2. Navigate to the local copy of your container image and upload it to the remote repository:
-```
-docker push apache/beam_python2.7_sdk
+docker push $USER-docker-apache.bintray.io/beam/python:beam_python2.7_sdk
```

Comment:
When I search the project for `bintray`, I only get a few matches:
```
buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy:          configuration.root = "${System.properties["user.name"]}-docker-apache.bintray.io/beam"
website/src/documentation/runtime/environments.md:./gradlew :sdks:python:container:py2:docker -Pdocker-repository-root=$USER-docker-apache.bintray.io/beam/python
website/src/get-started/wordcount-example.md:                      --worker_harness_container_image=apache-docker-beam-snapshots-docker.bintray.io/beam/go:20180515
website/src/get-started/wordcount-example.md:            --worker_harness_container_image=apache-docker-beam-snapshots-docker.bintray.io/beam/go:20180515
```
Of which it looks like the latter two are outdated.

Seeing as Bintray no longer has any special significance, maybe we should change the example repo to Docker hub. e.g. `docker push $YOUR_REPO:beam_python2.7_sdk`

Reply:
Updated it. PTAL when you have time.